### DISCUSSION TITLE
JSON response is not getting parsed

### DISCUSSION THREAD

**Discussion by rasulrukhilloev** [mockoon/mockoon#623] [Help] [answered]
JSON object is only getting parsed when it contains array. I explained in Stackoverfow extensively: https://stackoverflow.com/questions/70630611/body-is-not-getting-parsed-in-get-method

What could be the reason?

**Comment by rasulrukhilloev** [comment, marked as answer]
Sorry, idk how to delete the question. It was due to the coma at the end of JSON file

**Reply by 255kb** [reply]
No problem, I was writing the same answer. Glad you found the solution! 🙂
